src/content-views/constants.js

```diff
@@ -48,17 +48,17 @@ export const COLORS = Object.freeze({
   REACT_RENDER_HOVER: '#2683E2',
   REACT_COMMIT: '#ff718e',
   REACT_COMMIT_SELECTED: '#FF5277',
-  REACT_COMMIT_HOVER: '#ff335f',
-  REACT_LAYOUT_EFFECTS: '#c49edd',
+  REACT_COMMIT_HOVER: '#ed0030',
+  REACT_LAYOUT_EFFECTS: '#c88ff0',
   REACT_LAYOUT_EFFECTS_SELECTED: '#934FC1',
   REACT_LAYOUT_EFFECTS_HOVER: '#601593',
-  REACT_PASSIVE_EFFECTS: '#c49edd',
+  REACT_PASSIVE_EFFECTS: '#d8aff4',
```

Comment:
Was it intentional for passive and layout effect colors to only differ when they're not hovered over?

Reply:
Ah, good catch. must've slipped through. No, they're supposed to be same.